code-reviews/recipes/Java.md

```diff
@@ -0,0 +1,30 @@
+# Java Code Reviews
+
+## Java Style Guide
+
+[CSE](../../CSE.md) developers generally follow the [Google Java Style Guide](https://google.github.io/styleguide/javaguide.html).
+
+## Code Analysis
+
+We strongly believe that consistent style increases readability and maintainability of a code base. Hence we are recommending analyzers to enforce consistency and style rules.
+
+We make use of [Checkstyle](https://github.com/checkstyle/checkstyle) using the [same configuration used in the Azure Java SDK](https://github.com/Azure/azure-sdk-for-java/blob/master/eng/code-quality-reports/src/main/resources/checkstyle/checkstyle.xml).  
+
+Also frequently used is [FindBugs](http://findbugs.sourceforge.net/) and [PMD](https://pmd.github.io/).
+
+## Automatic Code Formatting
+
+Eclipse, and other Java IDEs, support automatic code formatting.  If using Maven, some developers also make use of the [formatter-maven-plugin](https://github.com/revelc/formatter-maven-plugin).
+
+## Build Validation
+
+It's important to enforce your code style and rules in the CI to avoid any team members merging code that does not comply with standards into your git repo.  If building using Azure DevOps, the [Maven and Gradle build tasks support](https://docs.microsoft.com/en-us/azure/devops/java/standalone-tools?view=azure-devops) using [PMD](https://pmd.github.io/), [Checkstyle](https://checkstyle.sourceforge.io/), and [FindBugs](http://findbugs.sourceforge.net/) code analysis tools as part of every build.
+
+## Code Review Checklist
+
+* [ ] Does the project use Lambda to make code cleaner?
+* [ ] Is dependency injection (DI) used?  Is it setup correctly?
+* [ ] If you're using Spring Boot, are you using @Inject instead of @Autowire?
```

Comment:
Nit: If you're using -> If the code uses Spring Boot 

since this is a code review check point

Reply:
Addressed in latest commit.